lisa/node.py

```diff
@@ -448,6 +448,23 @@ def get_working_path(self) -> PurePath:
 
         return self.get_pure_path(result.stdout)
 
+    def find_partition_with_freespace(self, size_in_gb: int) -> str:
```

Comment:
What's the reason it's copied here? The method exists on node already.

Reply:
This may be a new pushed one so it was not rebased before I pushed fixes? I'll do rebase again before pushing